server/package.json

```diff
@@ -19,7 +19,7 @@
     "@sentry/node": "^6.2.5",
     "@sentry/tracing": "^6.2.5",
     "axios": "^0.21.2",
-    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#v8.1.0",
+    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#td-reuse-twilio-numbers",
```

Comment:
Reminder to update this after https://github.com/bravetechnologycoop/brave-alert-lib/pull/48 is merged

Reply:
Done